pl-PL/Happy Birthday/Happy Birthday - notes.md

```diff
@@ -1,30 +1,31 @@
 ---
-title: Happy Birthday — Notes for Club Leaders
-language: en-GB
+title: Happy Birthday — Informacje dla prowadzącego
+language: pl-PL
 embeds: "*.png"
 materials: [""]
 ...
 
-#Introduction:
-In this project, children will be introduced to HTML tags & CSS properties. They will also edit tags and properties to make their own customised greeting card.
+# Wstęp
+Wykonując ten projekt dzieci poznają tagi HTML i właściwości CSS. Podczas zajęć będą także zmieniać tagi i właściwości, aby stworzyć swoją własną kartkę okolicznościową.
 
-#Online Resources
+# Materiały dostępne online
 
-We recommend using [trinket](https://trinket.io/) to write HTML & CSS online. This project contains the following trinkets:
+Do pisania kodu HTML i CSS online polecamy serwis [trinket](https://trinket.io/). Ten projekt zawiera następujące trinkety (fragmenty kodu):
 
-+ ['Intro to HTML and CSS' starting point](https://trinket.io/html/850a678202)
-+ ['Birthday Card' starting point  -- jumpto.cc/web-card](http://jumpto.cc/web-card)
++ ['Wstęp do HTML i CSS' - punkt początkowy](https://jumpto.cc/html-wstep)
++ ['Kartka urodzinowa' - punkt początkowy -- jumpto.cc/kartka-urodzinowa](http://jumpto.cc/kartka-urodzinowa)
 
-Children can also make use of this blank trinket [(jumpto.cc/trinket-blank)](http://jumpto.cc/trinket-blank) to write their own HTML & CSS, or alternatively they can use this template trinket [(jumpto.cc/trinket-template)](http://jumpto.cc/trinket-template).
+Do pisania swojego kodu HTML i CSS dzieci mogą też skorzystać z pustego trinketu [(jumpto.cc/trinket-blank)](http://jumpto.cc/trinket-blank) lub skorzystać szablonu: [(jumpto.cc/trinket-template)](http://jumpto.cc/trinket-template).
 
-There is also a trinket containing a sample solution to the challenges:
+Dostępny jest także trinket, który zawiera przykładowe rozwiązanie zadań:
 
-+ ['Birthday Card' Finished](https://trinket.io/html/e996dc0380)
++ [Skończona 'Kartka urodzinowa'](https://trinket.io/html/e996dc0380)
 
-#Offline Resources
-This project can be [completed offline](../html-css.html) if preferred. You can access the project resources by clicking the 'Download Project Materials' link for this project. This link contains a 'Project Resources' folder, which includes resources that children will need to complete this project offline. Make sure that each child has access to a copy of these resources. This folder includes the following files:
+# Materiały dostępne offline
+Projekt można także [wykonać offline](../html-css.html). Materiały do projektu można pobrać klikając w link "Pobierz materiały" na stronie z listą projektów. Pobrany plik zawiera katalog "Project Resources", który zawiera zasoby, których będą potrzebować dzieci by wykonać ten projekt offline. Upewnij się, że każde dziecko ma dostęp do tych materiałów. Katalog zawiera następujące pliki:
```

Comment:
Moim zdaniem "completed" jako "skończyć" tutaj nie pasuje. W sytuacji, kiedy rozpocznie się projekt online, to materiały dostępne offline nie mają zmian, których dokonało się online... Mam nadzieję, że jasno się wyraziłem ;)